pkg/plugins/plugins.go

```diff
@@ -75,7 +73,7 @@ type JSONData struct {
 	ID           string       `json:"id"`
 	Type         Type         `json:"type"`
 	Name         string       `json:"name"`
-	Alias        string       `json:"alias,omitempty"`
+	AliasIDs     []string     `json:"aliasIDs,omitempty"`
```

Comment:
In json this should probably be camelCase, .e.g. `aliasIds`. That's why I suggest to name it Aliases/aliases instead to simplify :)

Reply:
ID is uppercase so the camel case should be `aliasIDs`?

I also first thought of `aliases` but I think it's more descriptive to use `aliasIDs` since it makes it clear that it's expecting plugin `ID`s not just any alias.

Reply:
I think addind ids is helpful -- I am personally totally agnostic on the capitalizatino strategy, but in go for sure IDs, in json/ts... Ids is likey fine and more consistent but 🤷🏻

Reply:
feel free to merge with what you prefer

Reply:
Yep feel free